src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -71,34 +72,49 @@ const Sidebar = ({ drawerOpen, setDrawerOpen }: SidebarProps) => {
     }
   ];
 
+  const handleMoveContent = () => {
+    if (moveContent) {
+      setDrawerOpen(false);
+    }
+    setMoveContent(!moveContent);
+  };
+
+  console.log('NEW BAR-----------------');
   return (
     <NERDrawer open={drawerOpen} variant="permanent">
-      <DrawerHeader>
-        <IconButton onClick={() => setDrawerOpen(false)}>
-          {theme.direction === 'rtl' ? <ChevronRight /> : <ChevronLeft />}
-        </IconButton>
-      </DrawerHeader>
-      <Divider />
       <Box
-        overflow={'auto'}
-        sx={{ overflowX: 'hidden' }}
-        display="flex"
-        flexDirection={'column'}
-        flex={1}
-        justifyContent={'space-between'}
+        onMouseLeave={() => {
+          if (!moveContent) {
+            setDrawerOpen(false);
+            setMoveContent(false);
+          }
+        }}
       >
-        <Box>
-          {linkItems.map((linkItem) => (
-            <NavPageLink {...linkItem} />
-          ))}
-          {<NavUserMenu open={drawerOpen} />}
-        </Box>
-        <Box justifyContent={drawerOpen ? 'flex-start' : 'center'}>
-          <Box marginLeft={1.1}>
-            <Typography marginLeft={1.1}>Sponsored By:</Typography>
-            <Box component="img" sx={{ height: 40 }} alt="Kaleidoscope Logo" src="/kaleidoscope-logo-lockup.svg" />
+        <DrawerHeader>
+          <IconButton onClick={() => handleMoveContent()}>{moveContent ? <ChevronLeft /> : <ChevronRight />}</IconButton>
+        </DrawerHeader>
+        <Divider />
+        <Box
+          overflow={'auto'}
+          sx={{ overflowX: 'hidden' }}
+          display="flex"
+          flexDirection={'column'}
+          flex={1}
+          justifyContent={'space-between'}
+        >
+          <Box>
+            {linkItems.map((linkItem) => (
+              <NavPageLink {...linkItem} />
+            ))}
+            {<NavUserMenu open={drawerOpen} />}
+          </Box>
+          <Box justifyContent={drawerOpen ? 'flex-start' : 'center'}>
```

Comment:
This content should stay at the bottom of the sidebar, not directly following after the links